common/src/main/proto/TransportMessages.proto

```diff
@@ -443,6 +443,7 @@ message PbHeartbeatFromApplication {
   repeated PbWorkerInfo needCheckedWorkerList = 5;
   bool shouldResponse = 6;
   int64 shuffleFallbackCount = 7;
+  map<string, int64> shuffleFallbackCounts = 8;
```

Comment:
Looks weird to keep two variables of the same purpose. You can calculate the total fallback count from the detailed fallback count.

Reply:
+1, remove the original one